.github/workflows/build_packages.yaml

```diff
@@ -448,32 +459,55 @@ jobs:
           - arm64
         build_elixir:
           - no_elixir
+        build_machine:
+          - aws-arm64
+          - ubuntu-20.04
+        exclude:
+        - arch: arm64
+          build_machine: ubuntu-20.04
+        - arch: amd64
+          build_machine: aws-arm64
         include:
-          - profile: emqx
+          - os: alpine3.14
+            profile: emqx
             otp: 24.1.5-4
             elixir: 1.13.2
             arch: amd64
             build_elixir: with_elixir
+            build_machine: ubuntu-20.04
 
     steps:
+    - uses: AutoModality/action-clean@v1
+      if: matrix.build_machine == 'aws-arm64'
     - uses: actions/download-artifact@v2
       with:
         name: source
         path: .
     - name: unzip source code
       run: unzip -q source.zip
+
     - uses: docker/setup-buildx-action@v1
-    - uses: docker/setup-qemu-action@v1
+
+    - name: load rocksdb cache
+      uses: actions/cache@v2
       with:
-        image: tonistiigi/binfmt:latest
-        platforms: all
+        path: source/_build/default/lib/rocksdb/
```

Comment:
@thalesmg could elixir build reuse the cache or not?

Reply:
Yes, we can do it like in [here](https://github.com/emqx/emqx/blob/534cef2b3e9f41a836d0de67fd6d104293e67a37/.github/workflows/build_packages.yaml#L313-L326).

Reply:
thanks, I forget I saw that before while doing rebase 😄